roles/download/defaults/main.yml

```diff
@@ -434,6 +436,12 @@ crun_checksums:
     1.3: c0955cf6d3d832c0249bbaa71ed235abb35b8ca45fe07f2bd4501a00afb9bdc4
     1.4: 8e8081562503308f39f571acfe94afc663816ea0cb8f922145e2aaf0991415d7
 
+youki_checksums:
+  arm: 0
```

Comment:
```suggestion
  arm:
     0.0.1: 0
```

Reply:
else `{{ youki_checksums[image_arch][youki_version] }}` below breaks deployment on `arm` and `arm64`